WcaOnRails/lib/relations.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Relations
+  def self.get_chain(wca_id1, wca_id2)
+    find_chain([[wca_id1]], [[wca_id2]])
+  end
+
+  def self.competitions_together(person1, person2)
+    Competition.find(person1.competition_ids & person2.competition_ids)
+  end
+
+  # Internal
```

Comment:
This is interesting. Is there a reason you didn't mark these as private? (I'm honestly not really sure how private static methods work in ruby, so maybe that's why). Also, I personally would not have written tests for these internal methods, I feel like it's straying into testing implementation details. This is probably a testing philosophy thing though, and if you found these tests useful, then I'm okay with having them!